RecoVertex/V0Producer/src/V0Fitter.h

```diff
@@ -47,18 +41,28 @@
 #include "Geometry/CommonDetUnit/interface/GeomDet.h"
 #include "Geometry/TrackerGeometryBuilder/interface/GluedGeomDet.h"
 #include "FWCore/Framework/interface/ConsumesCollector.h"
-
 #include "DataFormats/TrackingRecHit/interface/TrackingRecHit.h"
 #include "DataFormats/BeamSpot/interface/BeamSpot.h"
-
-#include <string>
-#include <fstream>
+#include "CommonTools/CandUtils/interface/AddFourMomenta.h"
```

Comment:
better have all the includes in the .cc file: based on the changes in this file, these extra includes are not needed at the class declaration step.